istioctl/pkg/install/pre-check.go

```diff
@@ -78,8 +78,8 @@ func installPreCheck(istioNamespaceFlag *string, restClientGetter resource.RESTC
 	fmt.Fprintf(writer, "\n")
 	fmt.Fprintf(writer, "Istio-existence\n")
 	fmt.Fprintf(writer, "-----------------------\n")
-	ns, _ := c.getNameSpace(*istioNamespaceFlag)
-	if ns != nil {
+	_, err = c.getNameSpace(*istioNamespaceFlag)
+	if err == nil {
```

Comment:
shouldn't it be `err != nil`?

Reply:
oh nm ... I understand.

Reply:
err being null means the `get` succeeded, meaning the namespace already exists